app/src/common/services/role/roleService.spec.ts

```diff
@@ -0,0 +1,342 @@
+namespace common.services.user {
+
+    describe('UserService', () => {
```

Comment:
Name space and name of tests need to be updated? 'common.services.role' and 'Role Service'?